scripts/qa_crowbarsetup.sh

```diff
@@ -5038,10 +5039,140 @@ function ping_fips
     done
 }
 
+function get_stack_fips
+{
+    local stack=$1
+    local fip_file=$2
+    local all_fips=$(mktemp)
+    local fip_uuids=$(openstack stack resource list $stack -f value -c physical_resource_id --filter type=OS::Neutron::FloatingIP)
+    openstack floating ip list -f value > $all_fips
+
+    for uuid in $fip_uuids
```

Comment:
Could you fix the indenting, please? (Also in `ping_stack_fips`)

Reply:
Uhm...what would you like me to fix here? This is at the correct level (4 spaces for the `for`, 8 spaces for the block). Or would you like to have the `do` on the same line as the `for`?